### DISCUSSION TITLE
How to clear-up "Top Repositories" section?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47924] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have been interacting with certain developers on GitHub and consequently, their repositories show-up in my "Top Repositories" section (as evident from the attached image).
![Screenshot from 2023-02-19 07-53-50](https://user-images.githubusercontent.com/88191595/219908845-acbe45cb-3fd1-4292-ad6c-6cb539aa6a4d.png)

I no longer engage with "vlime/vlime", "hboetes/mg", "phoe-trash/clr2" and would prefer them not showing-up.

Is there any way to clear-up that section? Maybe some way to scrub away the past!

Thank you.

**Comment by melvincwng** [comment]
Hey @kathe, based on my understanding, the repositories present in the 'Top Repositories' section are automatically generated based on your GitHub activity.

So I don't think there's a way to manually clear those repositories in that section.

Perhaps, over time, as you interact with other repositories, the new repositories you interact with might take the place of those old repositories (which you no longer engage in). That's the only solution I can think of at the moment.

Hope this clarifies and answers your question 😃

**Reply by izmyname** [reply]
> 'Top Repositories' section are automatically generated

GitHub design is a mess, though I'm not surprised. After reading [this](https://github.com/orgs/community/discussions/10369) I'm not even sure if the people, responsible for GitHub design are sane, at all.

**Reply by rubengmurray** [reply]
I probably should have posted [this](https://github.com/orgs/community/discussions/110596#discussioncomment-11983526) here 👍 

It's terrible.
